pkg/services/sqlstore/team.go

```diff
@@ -312,10 +311,9 @@ func (ss *SQLStore) GetTeamsByUser(ctx context.Context, query *models.GetTeamsBy
 		var sql bytes.Buffer
 
 		sql.WriteString(getTeamSelectSQLBase([]string{}))
-		sql.WriteString(` INNER JOIN team_member on team.id = team_member.team_id`)
 		sql.WriteString(` WHERE team.org_id = ? and team_member.user_id = ?`)
 
-		err := sess.SQL(sql.String(), query.OrgId, query.UserId).Find(&query.Result)
+		err := sess.SQL(sql.String(), query.UserId, query.OrgId, query.UserId).Find(&query.Result)
```

Comment:
Change in behavior, the user permission (Admin, Member) will now be returned. Not sure if we want this.